src/components/App/App.jsx

```diff
@@ -3,14 +3,15 @@ import React from 'react';
 import Header from '../Header/Header.jsx'
 import './App.css';
 
+// Import the modules from here
+import MainComponent from '../MainGrocItemContainer/GrocItemMainComponent.jsx';
```

Comment:
No good Colin, check the name. Should be GrocItemMainComponent

Reply:
Went back and fixed the typo. Should be good now.